automated/linux/ltp/ltp.yaml

```diff
@@ -49,10 +49,3 @@ run:
         - cd ./automated/linux/ltp/
         - ./ltp.sh -T "${TST_CMDFILES}" -s "${SKIP_INSTALL}" -v "${LTP_VERSION}" -M "${TIMEOUT_MULTIPLIER}" -R "${ROOT_PASSWD}" -b "${BOARD}" -g "${BRANCH}" -e "${ENVIRONMENT}" -S "${SKIPFILE}"
         - ../../utils/send-to-lava.sh ./output/result.txt
-
-parse:
```

Comment:
I think we are safe to remove the parse pattern here. Objections?

Reply:
@chase-qi I agree. We are not using this parse.